api/v1alpha7/openstackmachinetemplate_types.go

```diff
@@ -36,6 +36,8 @@ type OpenStackMachineTemplate struct {
 	Spec OpenStackMachineTemplateSpec `json:"spec,omitempty"`
 }
 
+// +genclient
+// +genclient:Namespaced
```

Comment:
This was not rendered: https://deploy-preview-1702--kubernetes-sigs-cluster-api-openstack.netlify.app/api/v1alpha7/api